### ISSUE TITLE
FolderSettings

### ISSUE THREAD

**Issue by diVineProportion** [mwittrien/BetterDiscordAddons#120]
Is it possible to / implement:

- Give option to remove the number of child folders indication listed in the top left corner of the parent folder
- " to show the total notification count from all child folders?

**Comment by mwittrien** [comment]
Child folders?

**Comment by FuzzySockets** [comment]
Yes, this is already implemented. Navigate to the plugins folder and find `ServerFolders.config.json` then search for the string `showCountBadge` and ensure the value is set to false.
